lib/membrane_rtc_engine/endpoints/hls/config/sink_bin_config.ex

```diff
@@ -0,0 +1,40 @@
+if Code.ensure_loaded?(Membrane.HTTPAdaptiveStream.Manifest) do
+  defmodule Membrane.RTC.Engine.Endpoint.HLS.SinkBinConfig do
+    @moduledoc """
+    Module representing Membrane.HTTPAdaptiveStream.SinkBin configuration for the HLS endpoint.
```

Comment:
Please generate docs and check whether modules and functions are linked correctly
```suggestion
    Module representing `Membrane.HTTPAdaptiveStream.SinkBin` configuration for the HLS endpoint.
```